libbeat/common/kubernetes/metadata_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
not me, but goimports + vscode

I've seen this before at other projects and is not usually a blocker, but can remove it if that's a concern of some sorts.